cargo-shuttle/src/lib.rs

```diff
@@ -1157,13 +1159,19 @@ impl Shuttle {
 
                         return Ok(CommandOutcome::DeploymentFailure);
                     }
+
                     if DEPLOYER_END_MESSAGES_GOOD
                         .iter()
                         .any(|m| log_item.line.contains(m))
                     {
                         debug!("received end message, breaking deployment stream");
                         break;
                     }
+
+                    if log_item.line.contains(DEPLOYER_START_RESPONSE) {
```

Comment:
Do we still need the `DEPLOYER_END_MESSAGES_GOOD` line with this?

Reply:
I think so. Those are other cases in which we can break the loop. Maybe I can add the new message in that vector though?

Reply:
Done.